lib/cocoapods/validator.rb

```diff
@@ -303,6 +303,7 @@ def tear_down_validation_environment
     # for all available platforms with xcodebuild.
     #
     def install_pod
+      use_frameworks = @use_frameworks || spec.attributes_hash.has_key?('module_name')
```

Comment:
I'm not sure if using module_map as the key decision to determining if we should lint as a framework makes sense. Explicitly defining a `module_name` will be rare as using the pod name would be the normal option.

I think we should also determine if the pod is Swift, and in that case use frameworks as that's how it will be integrated.